src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision.Tests/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Tests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Tests</AssemblyName>
-    <VersionPrefix>1.0.2-preview</VersionPrefix>
```

Comment:
The test project is never published, we recommend setting the version to `1.0.0` and never change it.

Reply:
Done